kit/src/elements/input/mod.rs

```diff
@@ -304,8 +309,14 @@ pub fn Input<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
         .and_then(|b| b.then_some("password"))
         .unwrap_or("text");
 
+    let multiline =
+        cx.props.allow_line_breaks.unwrap_or(false) && !cx.props.is_password.unwrap_or(false);
```

Comment:
A bool, by default, would be `false` so unless those would be `true` by default (thus requiring an `unwrap_or` or similar to be set to `true` if those fields are `None`), its best to use `unwrap_or_default`

```suggestion
        cx.props.allow_line_breaks.unwrap_or_default() && !cx.props.is_password.unwrap_or_default();
```